### PR TITLE
forms: update example for multi-file upload

### PR DESCRIPTION
**Pull request by masenf**
Multi-file upload support was added in pynecone-io/pynecone#712.

* Remove sentence saying that only single file upload is supported
* Add sentence explaining the an event handler argument with `list[UploadFile]` annotation will be used to process multi-upload
* Update (and test) upload example 2 to allow uploading multiple images, and displaying them all in a grid.
  * Include "accepts" prop in the example, since it's not clear from the docs what format is expected. (This can be updated over the in main pynecode repo eventually)

### CODE REVIEW

**Comment by masenf** [comment]
CC: @Alek99 @picklelo @ElijahAhianyo who were involved in the feature.

File: pcweb/pages/docs/component_lib/forms.py [hunk 1]
```diff
@@ -821,9 +838,6 @@ def render_upload():
             width="100%",
             padding_bottom="1em",
         ),
-        doctext(
-            "Currently we only support one file upload, but we plan to support multiple file uploads in the future. ",
-        ),
         doctext(
             "Your event handler should be an async function that accepts a single argument, ",
             pc.code("file"),
```
Comment (ElijahAhianyo) [inline pcweb/pages/docs/component_lib/forms.py:843]: we've changed the syntax of the file uploads to use lists only for uniformity reasons. This means single uploads would use multi-upload syntax(`list[UploadFile]`). In the case of single uploads, the `multiple prop` should be set to `False`.

more [here](https://github.com/pynecone-io/pynecone/pull/811)

**Review by ElijahAhianyo — CHANGES_REQUESTED** [review]
Nice. Thanks for the addition. Just some minor comments about syntax

File: pcweb/pages/docs/component_lib/forms.py (hunk 1 above)
Comment (masenf) [reply pcweb/pages/docs/component_lib/forms.py]: Thanks. Fixed.

**Review by ElijahAhianyo — APPROVED** [review]
lgtm

**Review by Alek99 — APPROVED** [review]
Nice thanks!

**Pull request merged**